lib/taskana-core-test/src/test/java/acceptance/task/create/CreateTaskAccTest.java

```diff
@@ -0,0 +1,967 @@
+        .isInstanceOf(InvalidArgumentException.class)
+        .hasMessageContaining("not matching the service level");
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_CreateTask_When_ObjectReferenceSystemAndSystemInstanceIsNull() throws Exception {
+    ObjectReference objectReference =
+        ObjectReferenceBuilder.newObjectReference()
+            .system(null)
+            .systemInstance(null)
+            .company(defaultObjectReference.getCompany())
+            .type(defaultObjectReference.getType())
+            .value(defaultObjectReference.getValue())
+            .build();
+    Task task = taskService.newTask(defaultWorkbasketSummary.getId());
+    task.setClassificationKey(defaultClassificationSummary.getKey());
+    task.setPrimaryObjRef(objectReference);
+    task.setOwner("user-1-2");
+
+    Task createdTask = taskService.createTask(task);
+
+    assertThat(createdTask).isNotNull();
+    assertThat(createdTask.getCreator()).isEqualTo("user-1-2");
+  }
+
+  @WithAccessId(user = "admin")
+  @WithAccessId(user = "taskadmin")
+  @TestTemplate
+  void should_CreateTask_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
+      throws Exception {
+    Task task = createDefaultTask();
+
+    Task createdTask = taskService.createTask(task);
+
+    assertThat(createdTask).isNotNull();
+    assertThat(createdTask.getCreator())
+        .isEqualTo(taskanaEngine.getCurrentUserContext().getUserid());
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ThrowException_When_CreatingTheSameTaskTwice() throws Exception {
+    Task task = createDefaultTask();
+    task.setExternalId("MyExternalId");
+    Task createdTask = taskService.createTask(task);
+    Instant expectedPlanned = moveForwardToWorkingDay(createdTask.getCreated());
+
+    assertThat(createdTask).isNotNull();
+    assertThat(createdTask.getCreator())
+        .isEqualTo(taskanaEngine.getCurrentUserContext().getUserid());
+    assertThat(createdTask.getName()).isEqualTo(defaultClassificationSummary.getName());
+    assertThat(createdTask.getPrimaryObjRef().getValue())
+        .isEqualTo(defaultObjectReference.getValue());
+    assertThat(createdTask.getExternalId()).isNotNull();
+    assertThat(createdTask.getCreated()).isNotNull();
+    assertThat(createdTask.getModified()).isNotNull();
+    assertThat(createdTask.getBusinessProcessId()).isNotNull();
+    assertThat(createdTask.getClaimed()).isNull();
+    assertThat(createdTask.getCompleted()).isNull();
+    assertThat(createdTask.getModified()).isEqualTo(createdTask.getCreated());
+    assertThat(createdTask.getPlanned()).isEqualTo(expectedPlanned);
+    assertThat(createdTask.getState()).isEqualTo(TaskState.READY);
+    assertThat(createdTask.getParentBusinessProcessId()).isNull();
+    assertThat(createdTask.getPriority()).isEqualTo(defaultClassificationSummary.getPriority());
+    assertThat(createdTask.isRead()).isFalse();
+    assertThat(createdTask.isTransferred()).isFalse();
+
```

Comment:
This assertions (including computing expectedPlanned) are unnececary - a test for task creation already exists. Could you please remove them?